wagtail_content_import/parsers/__init__.py

```diff
@@ -15,4 +15,18 @@ def get_google_parser():
     """
     parser_string = get_google_parser_string()
 
-    return import_string(parser_string)
\ No newline at end of file
+    return import_string(parser_string)
+
+
+def get_docx_parser_string():
+    """
+    Get the dotted ``app.Model`` name for the Google Docs parser as a string.
```

Comment:
"Google Docs parser" - should this be "Office Open XML parser"?

Reply:
Thanks for catching that - I've corrected now.